superset/assets/javascripts/dashboard/components/SliceHeader.jsx

```diff
@@ -129,7 +130,7 @@ class SliceHeader extends React.PureComponent {
                   <i className="fa fa-pencil" />
                 </TooltipWrapper>
               </a>
-              <a className="exportCSV" href={this.props.exportCSVUrl}>
+              <a className="exportCSV" onClick={() => (this.props.exportCSV(slice))}>
```

Comment:
ideally these would be bound to the component but I see there is a precedent in other calls to create a new function on every render.

Reply:
fixed. now i use bound callbacks.